descriptors/ModuleDescriptor-template.json

```diff
@@ -89,7 +89,7 @@
     },
     {
       "id": "request-storage",
-      "version": "2.4",
+      "version": "2.5",
```

Comment:
This now needs to be `2.6` as [recent changes](https://github.com/folio-org/mod-circulation-storage/commit/6ae2901d2d78387b8c5a1a9b332bc5509ce2372f) already published `2.5`

Also, please update the version in the RAML documentation as well - https://github.com/folio-org/mod-circulation-storage/blob/2695111751afd5e153db60ff7f1ff2cb7c10aa05/ramls/request-storage.raml#L3